### PR TITLE
Update libstarlark from upstream bazel (github.com/bazelbuild/bazel)

### PR DESCRIPTION
**Pull request by mahmoudimus**

### CODE REVIEW

**Comment by mahmoudimus** [comment]
@aslepakurov this is ready for merge I believe. The failing tests will be fixed in #5

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "/Users/mahmoud/src/sbxscripting/bazelbuild/starlarky/.tmp/bazel"]
+	path = /Users/mahmoud/src/sbxscripting/bazelbuild/starlarky/.tmp/bazel
```
Comment (aslepakurov) [inline .gitmodules:2]: `mahmoud` ?

Comment (aslepakurov) [reply .gitmodules:2]: @mahmoudimus

Comment (mahmoudimus) [reply .gitmodules:2]: Oh hmm, yes, I need to keep this relative path, instead of absolute. Does that work?

Comment (aslepakurov) [reply .gitmodules:2]: yeap, that should work

**Review by aslepakurov — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (aslepakurov) [reply .gitmodules:2]: I will be merging this, can you please fix it in #5?

**Pull request merged**